### PR TITLE
feat: adds up notification in room

### PR DESCRIPTION
**Pull request by imGok**
- Adds up notifications count instead of number of unread rooms 
- Change the color of invite badge 
demo_notif

### CODE REVIEW

**Comment by hoangdat** [comment]
I saw in your demo, counter come from 11 to 14. Can you explain why?

**Comment by sherlockvn** [comment]
> * Adds up notifications count instead of number of unread rooms
> * Change the color of invite badge
>   demo_notif

I

**Comment by hoangdat** [comment]
Count = invitation + unread message 
Dont count = events (joining, accepting, ....)

File: lib/widgets/unread_rooms_badge.dart [hunk 1]
```diff
@@ -31,7 +32,14 @@ class UnreadRoomsBadge extends StatelessWidget {
             .rooms
             .where(filter)
             .where((r) => (r.isUnread || r.membership == Membership.invite))
-            .length;
+            .map((element) {
+          if (element.isUnread) {
+            return element.notificationCount;
+          } else {
+            return 1;
+          }
```
Comment (hoangdat) [inline lib/widgets/unread_rooms_badge.dart:40]: Deserve a method and avoid indent

**Comment by imGok** [comment]
- Put the correct badge size 
- Fix on bug on main with mention
image

**Comment by hoangdat** [comment]
Screenshot 2023-06-06 at 20 44 08

So our application as a student project, not a professional product!!!

**Comment by imGok** [comment]
It's from the mockup in figma made by professional designers :/
image

**Review by hoangdat — APPROVED** [review]

**Pull request merged**